src/analysis/seminorm.lean

```diff
@@ -416,6 +484,51 @@ end
 end module
 end normed_linear_ordered_field
 
+section seminorm_sup
+
+noncomputable theory
+
+variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E] [semi_normed_space ℝ 𝕜]
+variables [module ℝ E]
+variables {ι : Type*} [decidable_eq ι]
+variables (p : ι → seminorm 𝕜 E)
+variables (ι' : finset ι)
+
+@[simp]
+lemma seminorm_sup_singleton (p : ι → seminorm 𝕜 E) (i : ι):
+  ({i} : finset ι).sup p = p i :=
```

Comment:
This lemma has nothing to do with seminorms! Presumably `simp` finds the general case lemma.

Reply:
removed